app/javascript/components/supplemental_application/mapProperties.js

```diff
@@ -2,6 +2,7 @@ import _ from 'lodash'
 import moment from 'moment'
 
 import { mapApplication } from '../propMappers'
+import { updateApplicationAction } from './actions'
 
 // NOTE: what kind of saleforce domain object is this one? Fed
```

Comment:
@fddayan This is a `Field_Update_Comment__c` Salesforce object - if you check out that object in the Workbench SOQL Explorer you can see the fields on it.

Reply:
Great thanks.